samples/csharp_dotnetcore/50.teams-messaging-extensions-search/TeamsAppManifest/manifest.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.5/MicrosoftTeams.schema.json",
   "manifestVersion": "1.5",
   "version": "1.0",
-  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "id": "58e165da-bdca-41ce-98aa-c5144531ef1d",
```

Comment:
Please change this back to <<YOUR-MICROSOFT-APP-ID>>

Reply:
Sure will update it